modules/apps/frontend-data-set/frontend-data-set-taglib/src/main/java/com/liferay/frontend/data/set/taglib/servlet/taglib/ClassicDisplayTag.java

```diff
@@ -158,6 +158,14 @@ public String getStyle() {
 		return _style;
 	}
 
+	public boolean isShowBulkActionsManagementBar() {
+		return _showBulkActionsManagementBar;
+	}
+
+	public boolean isShowBulkActionsManagementBarActions() {
+		return _showBulkActionsManagementBarActions;
+	}
+
```

Comment:
This is the reason why I'm not a fan of booleans that start with `show*`. A boolean should ideally be named with adjective on end, in this case something like `bulkActionsManagementBarVisible`, that way getter sounds sane, `is...Visible`. But I guess it fine in this case as:

1. It is consistent with other existing toggle flags. If we change pattern with new one, we should change them all.
2. When there are a lot of UI toggle flags like in FDS, there is something to be said about keeping them visually grouped. As props/attributes are sorted alphabetically, `show*` are grouped and `*Visible` or `*Enabled` would not be.

So all in all, :+1: